lib/ansible/module_utils/basic.py

```diff
@@ -727,10 +727,13 @@ def jsonify(data, **kwargs):
     raise UnicodeError('Invalid unicode encoding encountered')
 
 
-def missing_required_lib(library):
+def missing_required_lib(library, reason=None):
     hostname = platform.node()
-    return "Failed to import the required Python library (%s) on %s's Python %s. Please read module documentation " \
-           "and install in the appropriate location." % (library, hostname, sys.executable)
+    msg = "Failed to import the required Python library (%s) on %s's Python %s." % (library, hostname, sys.executable)
+    if reason:
+        msg += "This is required %s." % reason
```

Comment:
needs a space before the quote here too

Reply:
Thanks for the pickup, will wait to see if the tests pass before making the chagnge

Reply:
Should be fixed now